Source/AwsCommonRuntimeKit/io/retryer/CRTAWSRetryToken.swift

```diff
@@ -7,17 +7,14 @@ public final class CRTAWSRetryToken {
     var rawValue: UnsafeMutablePointer<aws_retry_token>
 
     public init(retryStrategy: CRTRetryStrategy, allocator: Allocator = defaultAllocator) {
-        let intPointer = UnsafeMutablePointer<Int>.allocate(capacity: 1)
-        intPointer.pointee = 1
-        let atomicVar = aws_atomic_var(value: UnsafeMutableRawPointer(intPointer))
-        let retryStrategyPtr = UnsafeMutablePointer<CRTRetryStrategy>.allocate(capacity: 1)
-        retryStrategyPtr.initialize(to: retryStrategy)
+        let intPointer: UnsafeMutableRawPointer = fromPointer(ptr: 1)
```

Comment:
Here's that atomic thing again. If Swift doesn't have atomics, we should make a wrapper in this library for a generic atomic<T> where T = integral type or pointer